src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double upperA; //Length of the upper base
+    private final double lowerB; // Length of the lower base
+    private final double height; // height
+
+    public IsoscelesTrapezoid(double upperA, double lowerB, double height) {
+        this.upperA = upperA;
+        this.lowerB = lowerB;
+        this.height = height;
+        setColor(new ColorSupplier().getRandomColor());
+    }
+
+    @Override
+    public void toDraw() {
+        System.out.println("Figure: isoscales trapezoid, area: " + obtainTheArea()
+                + " sq. units, length of the upper base: " + upperA
+                + " units, length of the lower base: " + lowerB
+                + " units, height: " + height + " units, color: " + getColor());
+    }
+
+    @Override
+    public double obtainTheArea() {
```

Comment:
The method name 'obtainTheArea' is not following Java naming conventions or the task description. Consider renaming it to 'getArea' to align with common naming patterns and the checklist requirements.